### DISCUSSION TITLE
Is it possible to read and update a PPTX file using PpptxGenJS?

### DISCUSSION THREAD

**Discussion by CDHDeveloper** [gitbrent/PptxGenJS#1120] [Q&A]
In an app I'm currently designing,, I'd like to allow the user to add (or update) a slide in a currently existing PPTX file. Essentially the system will open the pre-existent file and add their slide to the batch. In my (admittedly brief) overview of PPTXGenJs it seems that only new creation is available. Is that true or have I overlooked something? If it is true, have you had others that have had similar needs and what advice did yo give them? 

Thanks in advance,
Chris

**Comment by gitbrent** [comment]
This library only generates new presentations.

Reading existing presentations is an existing item on the [Unimplemented Features](https://github.com/gitbrent/PptxGenJS/wiki/Unimplemented-Features) page.
